nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy

```diff
@@ -91,12 +91,9 @@ class ConanProxyRecipe
   ViewFacet configure(ConfigurableViewFacet facet) {
     Router.Builder builder = new Router.Builder()
 
-    conanApiV1.create(builder);
+    addBrowseUnsupportedRoute(builder)
 
-    builder.route(new Route.Builder()
-        .matcher(BrowseUnsupportedHandler.MATCHER)
```

Comment:
Minor: the `BrowseUnsupportedHandler` import at the top of this file can also be removed

Reply:
Thanks, fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/92/commits/522a5008e8f1db2847743259d68d93f144979ee6